src/Core/src/Platform/iOS/MauiImageView.cs

```diff
@@ -7,18 +7,34 @@ namespace Microsoft.Maui.Platform
 {
 	public class MauiImageView : UIImageView
 	{
+		WeakReference<ImageHandler>? _handler;
```

Comment:
Maybe this can be a public property? This way we don't have to change API at all much - the event still works - we just don't use it? Sort of like the iOS bending of a weak delegate OR an event? We can still use the existing iOS constructors.

Reply:
We can mark the bad one as obsolete just to let people know why - and us. Rather than just "it will be removed", maybe we can be "Use Handler instead. iOS platform views should not have strong references to handlers or cross-platform views."